internal/preflight/preflight.go

```diff
@@ -153,3 +176,31 @@ func (p *preflightHelper) verifyBuild(ctx context.Context,
 	}
 	return false, "Waiting for build verification"
 }
+
+func (p *preflightHelper) verifySign(ctx context.Context,
+	pv *kmmv1beta1.PreflightValidation,
+	mapping *kmmv1beta1.KernelMapping,
+	mod *kmmv1beta1.Module) (bool, string) {
+	log := ctrlruntime.LoggerFrom(ctx)
+
+	previousImage := ""
+	if module.ShouldBeBuilt(mod.Spec, *mapping) {
+		previousImage = module.IntermediateImageName(mod.Name, mod.Namespace, mapping.ContainerImage)
+	}
+
+	// at this stage we know that eiher mapping Sign or Container sign are defined
```

Comment:
Why is that?

Reply:
are you talking about the comment? Because if we entered the verifySign function, then the verification that either Module or Mapping sign sections are present, has been done above